docs/source/guide/pec-4-low-level.myst

```diff
@@ -32,7 +32,7 @@ The inference step of PEC is based on the Monte Carlo estimation protocol discus
 
 As shown in the Section [How do I use PEC?](pec-1-intro.myst), the standard way of applying PEC in Mitiq is based
 on the function {func}`.execute_with_pec()`. However, one may be interested in applying PEC with a lower-level of
-abstraction for reasons of efficiency or for customization proposes. 
+abstraction for reasons of efficiency or for customization purposes. 
```

Comment:
I know this PR is specific for ZNE but I found this minor typo in PEC docs. I can create a separate PR if that's preferred.

Reply:
Thanks for spotting the typo. It is probably better to use a different PR since this already involves a lot of different zne files.